### DISCUSSION TITLE
Icon needed for user settings

### DISCUSSION THREAD

**Discussion by johannaweintraub** [twilio-labs/paste#882] [Request: Icon] [answered]
Can we please add User Settings icon to paste? from streamline: Users/Geomertic Close Up Single User Neutral

![streamline-icon-single-neutral@140x140](https://user-images.githubusercontent.com/6243878/97040896-84252e80-1523-11eb-907b-15c0e04f2e65.png)

**Comment by serifluous** [comment, marked as answer]
Yep, Priyanka requested it here, too: #870 . We'll be adding it in our next sprint.

**Comment by johannaweintraub** [comment]
Ok, we can close this and just use the other icon that Priyanka requested.
